www/android/inappbrowser.js

```diff
@@ -47,7 +47,7 @@
 
         me.isHidden = function(){
             return hidden;
-        }
+        };
```

Comment:
revert these - the intent is to keep these as close to the original we forked from as possible.

Reply:
Oh okay, this causes the test to fail mind.